__tests__/helper.test.ts

```diff
@@ -3,6 +3,8 @@ import { ExternalResource } from '../src/ExternalResource';
 
 const blAvAuthNew = require('./fixtures/bl-av-auth.json');
 const blAvAuth = require('./fixtures/prev-auth.json');
+// const nwRightsLogoReqStatement = require('./fixtures/nw-rights-logo-reqStatement.json')
```

Comment:
Looks like this second manifest is never used. Should you delete this line and the associated fixture, or do you plan to add another test using this one?

Reply:
Thanks Demian I've removed that file and this line